charts/karmada/templates/_helpers.tpl

```diff
@@ -390,10 +390,15 @@ Return the proper karmada kubectl image name
 {{ include "common.images.image" (dict "imageRoot" .Values.kubectl.image "global" .Values.global) }}
 {{- end -}}
 
-{{- define "karmada.controllerManager.featureGates" -}}
-     {{- if (not (empty .Values.controllerManager.featureGates)) }}
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Is this necessary? or redundant?

Reply:
Is't redundant content.

had removed it.